docs/development.md

```diff
@@ -39,23 +39,33 @@ In addition, the command will also *minify* the JavaScript files to reduce the s
 
 #### Starting the dev server
 
-Run the following command:
+Run the following command and wait until the task exits with a `BUILD SUCCESSFUL`:
 
 ```sh
 ./gradlew appengineRun
 ```
+Run the command below instead if you want the console output to be visible: 
+```sh
+./gradlew appengineRun -Pdisable_daemon
+```
 
-Wait until the task exits with a `BUILD SUCCESSFUL`.
 The dev server URL will be `http://localhost:8888` as specified in `build.gradle`.
 
 #### Stopping the dev server
 
-Run the following command:
+If you start the server with ```./gradlew appengineRun```
+
+Run the following command 
 
 ```sh
 ./gradlew appengineStop
 ```
 
+If you start the server with ```./gradlew appengineRun -Pdisable_deamon```
+
+Press ```Ctrl+C``` will prompt the options to stop the server and type ```Y``` to terminate the server
```

Comment:
This does not seem to be universal - on my Mac, the second step is not required.